config/galaxy.ini.sample

```diff
@@ -213,6 +213,19 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # if it cannot find a local copy and conda_exec is not configured.
 #conda_auto_init = False
 
+# Certain dependency resolvers (namely conda) take a considerable amount of
+# time to build an isolated job environment in the job_working_directory.  Set
+# the following option to true to cache the dependencies in a folder. This
+# option is beta and should only be used if you experience long waiting times
+# before a job is actually submitted to your cluster. If you activate this
+# option, and you install new dependencies you may need to clear out old cached
+# environments
+#use_cached_dependency_manager = False
+
+# By default the tool_dependency_cache_dir is the _cache directory
+# of the tool dependency directory
+#tool_dependency_cache_dir = tool_dependency_dir/_cache
```

Comment:
I guess you saw my comment before I edited it to add quotes, GitHub hides everything between the angle brackets.
Anyway, I suggested to use

`<tool_dependency_dir>/_cache`